Windows/Debugger/CtrlDisAsmView.cpp

```diff
@@ -1377,3 +1542,50 @@ u32 CtrlDisAsmView::getInstructionSizeAt(u32 address)
 	u32 next  = manager.getNthNextAddress(start,1);
 	return next-address;
 }
+
+int regStringToMIPSGPReg(const char* name) {
+	std::string Name = name;
```

Comment:
you may just change the parameter type to `std::string` (or better` const std::string &`), if you pass a `const char*`
to a method asking for a `std::string` it will be implicit converted anyway.

Reply:
ah thanks, i was having trouble figuring out how to directly pass an std::string as an argument to a function.